experiments/PlaywrightSharp/PlaywrightSharp/ApiSurface/IBrowser.cs

```diff
@@ -0,0 +1,224 @@
+		/// </summary>
+		Task CloseAsync();
+	
+		/// <summary>
+		/// <para>
+		/// Returns an array of all open browser contexts. In a newly created browser, this
+		/// will return zero browser contexts.
+		/// </para>
+		/// </summary>
+		dynamic GetContexts();
+	
+		/// <summary><para>Indicates that the browser is connected.</para></summary>
+		bool IsConnected();
+	
+		/// <summary><para>Creates a new browser context. It won't share cookies/cache with other browser contexts.</para></summary>
+		/// <param name="acceptDownloads">
+		/// Whether to automatically download all the attachments. Defaults to <c>false</c>
+		/// where all the downloads are canceled.
+		/// </param>
+		/// <param name="bypassCSP">Toggles bypassing page's Content-Security-Policy.</param>
+		/// <param name="colorScheme">
+		/// Emulates <c>'prefers-colors-scheme'` media feature, supported values are `'light'`,
+		/// `'dark'`, `'no-preference'`. See <see cref="IPage.EmulateMediaAsync"/> for more
+		/// details. Defaults to '`light</c> .
+		/// </param>
+		/// <param name="deviceScaleFactor">Specify device scale factor (can be thought of as dpr). Defaults to <c>1</c></param>
+		/// <param name="extraHTTPHeaders">
+		/// An object containing additional HTTP headers to be sent with every request. All
+		/// header values must be strings.
+		/// </param>
+		/// <param name="geolocation">
+		/// </param>
+		/// <param name="hasTouch">Specifies if viewport supports touch events. Defaults to false.</param>
+		/// <param name="httpCredentials">Credentials for [HTTP authentication](https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication).</param>
+		/// <param name="ignoreHTTPSErrors">Whether to ignore HTTPS errors during navigation. Defaults to <c>false</c></param>
+		/// <param name="isMobile">
+		/// Whether the <c>meta viewport` tag is taken into account and touch events are enabled.
+		/// Defaults to `false</c>  Not supported in Firefox.
+		/// </param>
+		/// <param name="javaScriptEnabled">Whether or not to enable JavaScript in the context. Defaults to <c>true</c></param>
+		/// <param name="locale">
+		/// Specify user locale, for example <c>en-GB`, `de-DE`, etc. Locale will affect `navigator.language`
+		/// value, `Accept-Language</c> request header value as well as number and date formatting
+		/// rules.
+		/// </param>
+		/// <param name="offline">Whether to emulate network being offline. Defaults to <c>false</c></param>
+		/// <param name="permissions">
+		/// A list of permissions to grant to all pages in this context. See <see cref="IBrowserContext.GrantPermissionsAsync"/>
+		/// for more details.
+		/// </param>
+		/// <param name="proxy">
+		/// Network proxy settings to use with this context. Note that browser needs to be launched
+		/// with the global proxy for this option to work. If all contexts override the proxy,
+		/// global proxy will be never used and can be any string, for example <c>launch({ proxy:
+		/// { server: 'per-context' } })</c>
```

Comment:
This example is typescript. Can we get around that?

Reply:
Yes, all documentation will eventually be migrated to be C# for the .NET port.